README.md

```diff
@@ -464,6 +464,13 @@ rescue JWT::DecodeError
 end
 ```
 
+or by passing JWK as a simple Hash
+
+```
+jwks = { keys: [{ ... }] } # keys needs to be Symbol
```

Comment:
I added #348 to accept string keys. If that gets merged, this will need an update.

Reply:
If it's okay for you I would merge this doc update. Then you can rebase your PR and add the new documentation information.